Assets/Scripts/EOSPeer2PeerManager.cs

```diff
@@ -306,6 +306,11 @@ public ProductUserId HandleReceivedMessages()
                 RequestedChannel = null
             };
             P2PHandle.GetNextReceivedPacketSize(ref getNextReceivedPacketSizeOptions, out uint nextPacketSizeBytes);
+            
+            if (nextPacketSizeBytes == 0)
+            {
+                return null;
+            }
```

Comment:
Same as my other comment, shouldn't lines 296-301 be moved to be _after_ line 313? Sort of a nit-picky detail, which is why I'm not requesting changes. But curious if there is something I've missed some detail.

Reply:
Yeah, you're right, same as above.